blaze/server/server.py

```diff
@@ -219,7 +209,7 @@ def run(self, port=DEFAULT_PORT, retry=False, **kwargs):
 
 
 @api.route('/datashape', methods=['GET'])
-@crossdomain(origin='*', methods=['GET'])
+@cross_origin(origins='*', methods=['GET'])
```

Comment:
I'm in agreement with you; this PR is scoped to just reproduce the current functionality with the `flask-cors` extension.  I'll make a separate feature request for making this configurable.